firefox-ios/Client/Frontend/Browser/WebView/TemporaryDocumentLoadingView.swift

```diff
@@ -0,0 +1,98 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+
+class TemporaryDocumentLoadingView: UIView, ThemeApplicable {
+    private struct UX {
+        static let loadingBackgroundViewCornerRadius: CGFloat = 12.0
+        static let loadingContainerViewSpacing: CGFloat = 8.0
+        static let loadingContainerViewSidePadding: CGFloat = 20.0
+        static let backgroundColorAlpha: CGFloat = 0.3
+    }
+
+    private let loadingBackgroundView: UIView = .build { view in
+        view.layer.cornerRadius = UX.loadingBackgroundViewCornerRadius
+    }
+    private let loadingContainerView: UIStackView = .build { view in
+        view.axis = .vertical
+        view.spacing = UX.loadingContainerViewSpacing
+    }
+    private let loadingView: UIActivityIndicatorView = .build { view in
+        view.style = .medium
+    }
+    private let backgroundView: UIVisualEffectView = .build { view in
+        view.effect = UIBlurEffect(style: .systemThinMaterial)
+    }
+    private let loadingLabel: UILabel = .build { view in
```

Comment:
nit; spacing between variables 👀 I'm not commenting on all, but to me it makes it easier to read the code with spacing
```suggestion
    }

    private let loadingLabel: UILabel = .build { view in
```